apps/infrastructure/src/modules/chatbot/dynamodb.tf

```diff
@@ -0,0 +1,28 @@
+# Chatbot queries
+
+module "dynamodb_chatbot_queries" {
+  source = "git::github.com/terraform-aws-modules/terraform-aws-dynamodb-table.git?ref=715399dbe24f6443820bf5de80f6100b35d56355" # v4.0.0
+
+  billing_mode                = "PAY_PER_REQUEST"
+  deletion_protection_enabled = true
+
+  name                           = "ChatbotQueries"
+  hash_key                       = "id"
+  range_key                      = "createdAt"
```

Comment:
What is the id going to contain? Is it a unique id of a message or the session id?

If it's not already clear, I think it may be beneficiary to write a very brief RFC about the data model and how the table is going to be queried. 

After that it can be easier to choose the index (or indices) to accomodate the biggest number of direct queries against full scans

Reply:
it's a unique id of a message.